sycl/include/sycl/ext/intel/experimental/esimd/detail/math_intrin.hpp

```diff
@@ -12,6 +12,8 @@
 
 /// @cond ESIMD_DETAIL
 
+#define ARG_UNUSED(x) (void)x
```

Comment:
Please
1) move this to some utility header
2) use __ESIMD_ prefix to avoid polluting user namespace
3) consider adding #ifndef __SYCL_DEVICE_ONLY__

Reply:
`ARG_USED` is removed as it is no longer used.